src/main/java/lotto/domain/LottoCalculator.java

```diff
@@ -0,0 +1,57 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lotto.dto.LottoWinResult;
+import lotto.enums.LottoCalEnum;
+import lotto.enums.LottoCriteria;
+
+public class LottoCalculator {
+
+    private static final Map<Integer, Integer> lottoCalMap = new HashMap<>();
+
+    static {
+        for(LottoCalEnum lottoCalEnum : LottoCalEnum.values()){
+            lottoCalMap.put(lottoCalEnum.getWinCount(),lottoCalEnum.getWinMoney());
+        }
+    }
+
+    public static int cal(int winCount, boolean isBonus){
```

Comment:
'축약하지 않는다'도 요구사항 중 하나였습니다 😮 
```suggestion
    public static int calculate(int winCount, boolean isBonus){
```

Reply:
이런.. 자꾸 제 안좋은 습관이 나오는것같아요 ㅠㅠ